src/test/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepositoryTest.java

```diff
@@ -129,6 +129,10 @@ private void jobSetup(ParameterDefinition... parameterDefinitions) {
         .thenReturn(Arrays.asList(parameterDefinitions));
   }
 
+  private void triggerSetup() {
```

Comment:
that's a lot of repetition, can you make it a `@Before` method?

Reply:
Done. Don't know why it didn't occur to me :smile: